WorkPackages/jupyter-ecosystem.tex

```diff
@@ -44,7 +45,12 @@
 Jupyter ecosystem projects with an emphasis on open science.
 
 repo2docker is a project for creating
-reproducible environments in which Jupyter notebooks can be run. BinderHub is software for operating a web service using repo2docker, which enables sharing of interactive and reproducible Jupyter (and Rstudio) environments on the web with a single link. We will develop repo2docker and BinderHub further to meet the needs of the open science community.
+reproducible environments in which Jupyter notebooks can be run.
```

Comment:
```suggestion
reproducible environments in which Jupyter notebooks (and other user interfaces) can be run.
```

is this meant to be a high-level overview (short and sweet with omissions) or is being pedantic (like the above suggestion) useful here?

Reply:
Pedantic is good.